data.json

```diff
@@ -1547,4 +1547,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/56837137/139258468-f7b162b5-94f2-4fb3-94e0-7da43db795dd.png",
     "issueNumber": 805
   }
+   {
```

Comment:
Please can you format line `1550`, so the brace is in line? @UltimateBrains 
After that, it should be good.